charts/aws-efs-csi-driver/values.yaml

```diff
@@ -66,6 +66,11 @@ controller:
     #   cpu: 100m
     #   memory: 128Mi
   nodeSelector: {}
+  updateStrategy: {}
+  # type: RollingUpdate
+  # rollingUpdate:
```

Comment:
What strategy did you use to speed up your deployments? Did you test both and compared the results? Could you maybe share it in the PR description please?

Reply:
I haven't done any test to compare speed between different strategies..
With default strategy the rollout take too long and helm times out after 300 second.
 We would like to use OnDelete. We are using only spot instances and nodes are replaced frequently.
(will update the description as well)

Reply:
Now I understand it better, so you tested DaemonSet `OnDelete` strategy only to get past the helm timeout. And the problem does not necessarily come from having huge number of nodes but could be caused by instance type of worker nodes, as you observed.

Then I think leaving controller deployment without any comment is ok because at this point we don't know what alternative to suggest (and controller pods were not causing the issue anyway) and instead it should go under node values - see my suggestions.